cpu/esp32/Kconfig.esp32s2

```diff
@@ -0,0 +1,277 @@
+# Copyright (c) 2020 HAW Hamburg
+#               2022 Gunar Schorcht
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config CPU_CORE_XTENSA_LX7
+    bool
+    select CPU_ARCH_XTENSA
+
+config CPU_CORE
+    default "xtensa-lx7" if CPU_CORE_XTENSA_LX7
```

Comment:
This is a duplicate from `Kconfig.esp32s3`, better move this in `Kconfig.common` or `esp_common/Kconfig`

Reply:
OK, I will place it in `cpu/esp32/Kconfig` which contains all the definitions that are common for all ESP32x SoCs. `cpu/esp32/Kconfig.common` contains only common ESP32x SoC menu entries and is included in the menu of the specific ESP32x SoC. Maybe it would have been better to call it `cpu/esp32/Kconfig.menu_common` or so.

May I squash the change directly to avoid an additional CI compilation cycle?

Reply:
Sure, squash away